js/ag_reference.js

```diff
@@ -784,6 +938,63 @@ AttributeGroupColumn.prototype = {
             return {isHTML: true, value: module._formatUtils.printMarkdown(formattedValue, { inline: true }), unformatted: formattedValue};
         }
         return {isHTML: false, value: formattedValue, unformatted: formattedValue};
+    },
+
+    /**
+     * @desc An array of objects that have `column` which is column name (this is different from other sortColumns),
+     * and `descending` which is true/fals. The `descending` boolean indicates whether we should change the direction of sort or not.
+     * The column name is going to be database column names (equivalent to this.term)
+     *
+     * - if sortable passed as false, it will be empty.
+     * - if baseColumn exists, it will return the column order of baseColumn
+     * - otherwise the current column
+     * @type {Object[]}
+     */
+    get _sortColumns() {
```

Comment:
I thought these functions were already available on the column object? 

Unless the body of determine sortable is different here, which would make sense why this is redefined.